solutions/ac-MultipleOf2019.mdx

```diff
@@ -2,11 +2,53 @@
 id: ac-MultipleOf2019
 source: AtCoder Beginner Contest
 title: Multiple of 2019
-author: Maggie Liu, Brad Ma, Mohammad Nour Massri
+author: Maggie Liu, Brad Ma, Mohammad Nour Massri, David Zhang
 ---
 
 [Official Analysis](https://img.atcoder.jp/abc164/editorial.pdf#page=10)
 
+## Explanation
+In order to store sections of the string, we can make a suffix array $S$, such that:
+
+$${S_i = s_n + 10*s_{n-1} +100*s_{n-2} + ... + {10^{n-i}} * s_{i}}$$
+
+For example, in the string $$1234$$ and the modulus of $$3$$, the suffix array would be
```

Comment:
> modulus of $$3$$

?

Reply:
What I mean is, 1234 is the string, 3 is the modulus